testRearrangeCarGroup8.py

```
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+

```

Another test worth doing is if one or both inputs given is/are the empty list!
How should we handle such case? If they are both empty, I would just raise the message "No moves required" or something like that. If just one is empty, we should raise an error, I believe.

---

I think that's handled by `test_empty_parking_lot` XD

---

Ok! I added a comment there now then, for clarification of terminology :)